File: app/src/main/java/com/globalwarming/earthsaver/directories/DirectoryEntry.java
package com.globalwarming.earthsaver.directories;

public class DirectoryEntry {

    private String id;

    private String note;

    private Boolean answer;

    private Long point;

    private Long timestamp;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getNote() {
        return note;
    }

    public void setNote(String note) {
        this.note = note;
    }

    public Boolean getAnswer() {
        return answer;
    }

    public void setAnswer(Boolean answer) {
        this.answer = answer;
    }

    public Long getPoint() {
        return point;
    }

    public void setPoint(Long point) {
        this.point = point;
    }

    public Long getTimestamp() {
        return timestamp;
    }

    public void setTimestamp(Long timestamp) {
        this.timestamp = timestamp;
    }
}

File: app/src/main/java/com/globalwarming/earthsaver/directories/DirectoryEntry.kt
package com.globalwarming.earthsaver.directories

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class DirectoryEntry(
    var id: String = "",
    var note: String = "",
    var answer: Boolean = false,
    var point: Long = 0L,
    var timestamp: Long = 0L
): Parcelable